src/app/src/components/FilterSidebar.jsx

```diff
@@ -131,7 +132,7 @@ class FilterSidebar extends Component {
                 case filterSidebarTabsEnum.guide:
                     return <FilterSidebarGuideTab />;
                 case filterSidebarTabsEnum.search:
-                    return <FilterSidebarSearchTab />;
+                    return <Route component={FilterSidebarSearchTab} />;
```

Comment:
This was done to have `history.push` available in the `mapDispatchToProps` function.